multinet/db.py

```diff
@@ -307,23 +308,31 @@ def aql_query(
 def create_graph(
     workspace: str,
     graph: str,
-    node_tables: List[str],
     edge_table: str,
     arango: ArangoClient,
+    from_vertex_collections: Set[str],
+    to_vertex_collections: Set[str],
```

Comment:
I think the `arango` argument should go last in the list (to match the way the other `@with_client` functions are).